ODROID-C2.md

```diff
@@ -0,0 +1,14 @@
+## Additional Packages
+
+* X.org video driver package: xf86-video-fbdev
+* Mali EGL and GLES drivers (currently framebuffer only): odroid-c2-libgl-fb
+* wiringPi modified for the ODROID-C1/C2: wiringc1
+
+## Display
+
+Configuring the resolution of the screen on the HDMI port is done through two items
+
+* /boot/boot.ini
+* /usr/bin/amlogic.sh (called by amlogic.service)
```

Comment:
End user configuration is not done through amlogic.sh, only boot.ini.  This is also mentioned in the installation instructions.